remotespark/livyclientlib/livyclientfactory.py

```diff
@@ -18,14 +18,14 @@ def __init__(self):
         self.logger = Log("LivyClientFactory")
         self.max_results = conf.max_results_sql()
 
-    def build_client(self, session):
+    def build_client(self, session, msg_printer):
         assert session is not None
         kind = session.kind
 
         if kind == Constants.session_kind_pyspark:
-            return PandasPysparkLivyClient(session, self.max_results)
+            return PandasPysparkLivyClient(session, msg_printer, self.max_results)
```

Comment:
Why are there no modification to `PandasPysparkLivyClient` and `PandasScalaLivyClient`?